trunk/NDHMS/OrchestratorLayer/config.f90

```diff
@@ -141,6 +141,8 @@ module config_base
      integer :: rtFlag
      integer ::khour
 
+     integer :: channel_loss_option = 0
```

Comment:
Is there a reason we don't assign the default values for other options here as well?  Just curious.

Reply:
The reason I assigned a default value here was so that existing namelists that don't explicitly specify a value won't produce unexpected effects. You're right, though: all options should have default values where an defensible default exists.

Reply:
Some of that (at least one) is probably my fault for blindly following.  Thanks for the clarification.